astropy/io/votable/tests/table_test.py

```diff
@@ -186,6 +186,18 @@ def test_empty_table():
     astropy_table = table.to_table()  # noqa
 
 
+def test_no_field_not_empty_table():
+    votable = parse(get_pkg_data_filename('data/no_field_not_empty_table.xml'))
+    table = votable.get_first_table()
+    assert len(table.fields) == 0
+    assert len(table.infos) == 1
+
+
+@raises(E25)
+def test_no_field_not_empty_table_exception():
+    votable = parse(get_pkg_data_filename('data/no_field_not_empty_table.xml'), verify='exception')
```

Comment:
It's a bit cleaner to use `pytest` here. `@raises` was written a long time ago before `pytest` has it.
```suggestion
def test_no_field_not_empty_table_exception():
    with pytest.raises(E25):
        votable = parse(get_pkg_data_filename('data/no_field_not_empty_table.xml'), verify='exception')
```